modules/openapi-generator/src/main/resources/go/model_oneof.mustache

```diff
@@ -1,14 +1,16 @@
 // {{classname}} - {{#description}}{{{description}}}{{/description}}{{^description}}struct for {{{classname}}}{{/description}}
 type {{classname}} struct {
 	{{#oneOf}}
-	{{{.}}} *{{{.}}}
+	{{#lambda.titlecase}}{{{.}}}{{/lambda.titlecase}} *{{{.}}}
```

Comment:
Do you have a mininal spec to show the issue related to this fix?

We may better fix it in the Go client generator (java class) instead.

Reply:
Yes, so the problem is if we have simple types in the oneOf (bool, float, string) as we generate

```
type UiNodeInputAttributesValue struct {
	bool    *bool
	float32 *float32
	string  *string
}
```

from

```
      "uiNodeInputAttributesValue": {
        "oneOf": [
          {
            "type": "string"
          },
          {
            "type": "number"
          },
          {
            "type": "boolean"
          }
        ]
      },
```

Using the upper case variant here fixes that:

```
type UiNodeInputAttributesValue struct {
	Bool    *bool
	Float32 *float32
	String  *string
}
```